client/components/autocomplete_list/index.coffee

```diff
@@ -0,0 +1,48 @@
+_ = require 'underscore'
+React = require 'react'
+Autocomplete = null
+{ label, input, div, button } = React.DOM
+
+module.exports = (el, props) ->
+  React.render React.createElement(AutocompleteList, props), el
+
+module.exports.AutocompleteList = AutocompleteList = React.createClass
+
+  getInitialState: ->
+    loading: true, items: @props.items or []
+
+  componentDidMount: ->
+    @addAutocomplete()
+
+  addAutocomplete: ->
+    Autocomplete ?= require '../autocomplete/index.coffee'
```

Comment:
I was having some trouble with typeahead in tests... there's definitely a better solution to this though.